pinot-core/src/main/java/org/apache/pinot/core/common/MinionConstants.java

```diff
@@ -61,4 +61,17 @@ private MinionConstants() {
     public static final String MERGE_TYPE_KEY = "mergeTypeKey";
     public static final String MERGED_SEGMENT_NAME_KEY = "mergedSegmentNameKey";
   }
+
+  public static class RealtimeToOfflineSegmentsTask {
+    public static final String TASK_TYPE = "realtimeToOfflineSegmentsTask";
+    // window
```

Comment:
Let's move the segment processing framework related configs to a static separate class. These configs will also be used by other tasks.

Reply:
Can we do that when we add these other tasks? It will read odd for this PR

Reply:
I see. I will make the changes with my merge & roll-up task generator PR.